core/src/scene/sceneLoader.cpp

```diff
@@ -330,6 +349,236 @@ void SceneLoader::loadTextures(Node textures, Scene& scene) {
+        logMsg("WARNING: animated property will be set but not yet implemented in styles\n"); // TODO
+        if (!animatedNode.IsScalar()) { logMsg("WARNING: animated flag should be a scalar\n"); }
+        else {
+            try {
+                style->setAnimated(animatedNode.as<bool>());
+            } catch(const BadConversion& e) {
+                logMsg("WARNING: Expected a boolean value in animated property. Using default (false).\n");
+            }
+        }
+    }
+
+	if (Node blendNode = styleNode["blend"]) {
+
+        std::string str = blendNode.as<std::string>();
+
+        if      (str == "none")     { style->setBlendMode(Blending::none); }
+        else if (str == "add")      { style->setBlendMode(Blending::add); }
+        else if (str == "multiply") { style->setBlendMode(Blending::multiply); }
+        else if (str == "overlay")  { style->setBlendMode(Blending::overlay); }
+        else if (str == "inlay")    { style->setBlendMode(Blending::inlay); }
+        else { logMsg("WARNING: invalid blend mode \"%s\"\n", str.c_str()); }
+
+    }
+
+    Node texcoordsNode = styleNode["texcoords"];
+    if (texcoordsNode) {
+
+        logMsg("WARNING: texcoords style parameter is currently ignored\n");
+
+        if (texcoordsNode.as<bool>()) { } // TODO
+        else { } // TODO
+
+    }
+
+    Node shadersNode = styleNode["shaders"];
+    if (shadersNode) {
+        loadShaderConfig(shadersNode, *(style->getShaderProgram()));
+    }
+
+    Node materialNode = styleNode["material"];
+    if (materialNode) {
+        loadMaterial(materialNode, *(style->getMaterial()), scene);
+    }
+
+    Node lightingNode = styleNode["lighting"];
+    if (lightingNode) {
+        std::string lighting = lightingNode.as<std::string>();
+        if (lighting == "fragment") { style->setLightingType(LightingType::fragment); }
+        else if (lighting == "vertex") { style->setLightingType(LightingType::vertex); }
+        else if (lighting == "false") { style->setLightingType(LightingType::none); }
+        else if (lighting == "true") { } // use default lighting
+        else { logMsg("WARNING: unrecognized lighting type \"%s\"\n", lighting.c_str()); }
+    }
+
+	Node textureNode = styleNode["texture"];
+    if (textureNode) {
+    	auto spriteStyle = dynamic_cast<SpriteStyle*>(style);
+        if (spriteStyle) {
+        	std::string textureName = textureNode.as<std::string>();
+            auto atlases = scene.spriteAtlases();
+            auto it = atlases.find(textureName);
+            if (it != atlases.end()) {
+            	spriteStyle->setSpriteAtlas(it->second);
+            } else {
+            	logMsg("WARNING: undefined texture name %s", textureName.c_str());
+            }
+        }
+	}
+
+    Node urlNode = styleNode["url"];
+    if (urlNode) { logMsg("WARNING: loading style from URL not yet implemented\n"); } // TODO
+
+}
+
+Node SceneLoader::propORing(const std::string& propStr, Node& styles, const MIXES& mixes) {
```

Comment:
Not crazy about the name propORing (keep reading it as O-ring), but also not sure what to call it. Maybe propFlag? Though I suppose there's also nothing inherent in having flags use OR behavior.